spv/sync.go

```diff
@@ -1016,19 +1031,40 @@ func (s *Syncer) startupSync(ctx context.Context, rp *p2p.RemotePeer) error {
 					}
 					s.loadedFilters = true
 				}
+				if atomic.CompareAndSwapUint32(&s.atomicWalletSynced, 0, 1) &&
+					s.notificationCallbacks.SyncUpdated != nil {
+					s.notificationCallbacks.SyncUpdated(true)
+				}
```

Comment:
add helper methods (e.g. `s.synced()` and `s.unsynced()`) for these events instead of repeating this throughout